ui/i18n/src/main/res/values/strings.xml

```diff
@@ -611,7 +612,9 @@
         <item quantity="one">1 hour</item>
         <item quantity="other">%d hours</item>
     </plurals>
-    <string name="auto_enable_label">Auto-enable</string>
+    <string name="auto_enable_label">Automatically activate the sleep timer when pressing play</string>
+    <string name="auto_enable_label_with_times">Automatically activate the sleep timer when pressing play at %s</string>
```

Comment:
"at" or "between %s and %s"? I'm not a native speaker but "at" sounds wrong when referring to a time range

Reply:
Done.